src/meta/src/barrier/progress.rs

```diff
@@ -312,8 +336,11 @@ impl CreateMviewProgressTracker {
                 backfill_upstream_types.insert(actor, backfill_upstream_type);
             }
 
+            let backfill_order_state = backfill_order_map.remove(&creating_table_id);
```

Comment:
Can you explain this line?

Reply:
Backfill orders are tracked per job. So we just extract the backfill order graph for a specific job (`creating_table_id`) and use it for recovery.